core/blockchain_snapshot_test.go

```diff
@@ -570,6 +570,7 @@ func TestLowCommitCrashWithNewSnapshot(t *testing.T) {
 // committed point so the chain should be rewound to genesis and the disk layer
 // should be left for recovery.
 func TestHighCommitCrashWithNewSnapshot(t *testing.T) {
+	skipForTrieDB(t)
```

Comment:
Why do these tests have to be skipped explicitly now?

Reply:
This is because this test relies on the behavior of the GC in the trie.
For example, in this example, there would be a chain
G->C1->C2->C3->C4->C5->C6->C7->C8 (HEAD)
Among them, the snapshot is persisted at C4, and then the trie commit at C6. The test expects that the restart after the crash will start from G, because there is no trie information in the DB in the range of C1-4. The current trie db is currently written directly, so C4 also exists in the DB, so this test case cannot pass.